File: src/golang/1_1_persist_data.go
package main

import (
	"os"
)

func SaveData1(path string, data []byte) error {
	fp, err := os.OpenFile(path, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0664)
	if err != nil {
		return err
	}
	defer fp.Close()
	_, err = fp.Write(data)
	return err
}

func main() {
	byteArray := []byte{96, 86}
	SaveData1("./test", byteArray)
}

File: src/python/1_1_persist_data.py
from typing import List

def saveData1(path:str, data: List[bytes]):
    with open(path, "wb") as file:
        file.write(data)

if __name__ == "__main__":
    byteArray = bytearray([96, 86])
    saveData1("./test_python", byteArray)